src/lib/Libtpp/tpp_client.c

```diff
@@ -673,7 +673,8 @@ tpp_init(struct tpp_config *cnf)
 		return -1;
 	}
 
-	TPP_DBPRT(("localhost = %s", tpp_conf->node_name));
+	snprintf(log_buffer, TPP_LOGBUF_SZ, "TPP leaf node names = %s", tpp_conf->node_name);
```

Comment:
Not required since it is a requirement. But no harm adding either. In the case you want to add a check, do it before the snprintf, so if tpp_conf->node_name, you can print an error message and return -1 from this tpp_init() function.